playwright.config.ts

```diff
@@ -23,7 +23,7 @@ const config: PlaywrightTestConfig<TestOptions> = {
   /* Retry on CI only */
   retries: process.env.CI ? 2 : 0,
   /* Opt out of parallel tests on CI. */
-  workers: process.env.CI ? 1 : 3,
+  workers: 1,
```

Comment:
Some of the tests reuse the same resources, running those tests in parallel will fail sometimes